parsl/tests/test_mpi_apps/test_mpi_mode_disabled.py

```diff
@@ -1,4 +1,3 @@
-import logging
```

Comment:
I have been trying to keep tidyups in separate PRs from functionality changes, with a goal of the changes being on-topic - this stuff is super confusing to debug when it turns out eg. something broke because of a changed import rather than a launcher change.

Reply:
I made this into PR #3436